app/src/main/java/com/github/se/cyrcle/ui/profile/CreateProfileScreen.kt

```diff
@@ -0,0 +1,51 @@
+package com.github.se.cyrcle.ui.profile
+
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material3.Scaffold
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.testTag
+import androidx.compose.ui.res.stringResource
+import com.github.se.cyrcle.R
+import com.github.se.cyrcle.model.user.UserViewModel
+import com.github.se.cyrcle.ui.navigation.NavigationActions
+import com.github.se.cyrcle.ui.theme.atoms.Text
+import com.github.se.cyrcle.ui.theme.molecules.TopAppBar
+
+@Composable
+fun CreateProfileScreen(navigationActions: NavigationActions, userViewModel: UserViewModel) {
+  val screenTitle = stringResource(R.string.profile_screen_title)
+
+  Scaffold(
+      modifier = Modifier.fillMaxSize().testTag("CreateProfileScreen"),
+      topBar = { TopAppBar(navigationActions, screenTitle) }) { padding ->
+        Column(
+            modifier = Modifier.fillMaxSize().padding(padding),
+            verticalArrangement = Arrangement.Center) {
+              Text("TODOOOOOOOOOO")
+
+              // GoogleSignInButton(
```

Comment:
Any reason why all this is commented? Is this a feature to roll out in a future PR?

Reply:
This was just to test but it cannot be un-commented as is, it does not make any checks for a user already existing in the database and does not fill in all the fields for the User. I left it there as an example and help for future work.

Reply:
Good call then (for next time maybe make it a block comment so it's easier to uncomment :)) )